cob_openni2_tracker/launch/body_tracker_nodelet.launch

```diff
@@ -21,7 +21,7 @@
 
 
 	<node pkg="nodelet" type="nodelet" name="body_tracker" args="load cob_openni2_tracker/body_tracker_nodelet /$(arg camera_name)/$(arg camera_name)_nodelet_manager" output="screen" cwd="ROS_HOME">
-		<param name="camera_frame_id" type="str" value="$(arg camera_name)_link" /> <!-- $(arg camera_name)_depth_optical_frame -->
+		<param name="camera_frame_id" type="str" value="$(arg camera_name)_rgb_frame" />   <!-- "$(arg camera_name)_link" -->
```

Comment:
I am currently working with ASUS. However, we are interested testing this with Microsoft Kinect v2 (Xbox One) which we have in hand. We are planning to test this since ASUS's sensor is discontinued we want to have latest sensor working with the project. 

So is Kinect v2 supposed to work with this package? If so, we can try and let you guys know if it does, but keep in mind will take a while since we have other priorities over this for now.